Add option to not block cover art with track info

Twinov opened the pull request:
I've added an option in the settings under Display to no longer have the track information (folder and title) block the cover art of the audio file being played. I'm really new to Android programming so apologies if I made any mistakes 😀. I've tested my changes on my Pixel 6 Pro running Android 12 and it works as expected.  
Here's some screenshots of the changes:
As it is right now:  
![image](https://user-images.githubusercontent.com/23278147/155436281-a9b40909-0462-46bb-bdac-483c520d04ad.png)
With the changes I added, you can have the cover art be below the picture:  
![image](https://user-images.githubusercontent.com/23278147/155436345-dd3b0c8b-604e-4d86-b259-fad4e8a34f6b.png)
![image](https://user-images.githubusercontent.com/23278147/155436363-1685063d-701a-4be9-b880-a40647023c6d.png)

By the way, unrelated to the PR, but it may be useful to add a `.gitignore` file to the repo. I used Android Studio and it manually adds a `.idea` folder so it would be useful to other contributors if there's a gitignore file with common folders/files like that.

flackbash commented:
That looks good, thanks :) I was a bit worried about smaller display sizes, but it seems to be working fine. I'll make some more tests and merge it if I don't find any problems.

There is a `.gitignore`, but only some files in the `.idea` directory are ignored. I'll iterate over it again though to check what should be included in the repo and what not.

Twinov commented:
Ah you're right there's a gitignore, I was being dumb, my bad. For reference, here's the files that are shown in my `git status` as changed but unstaged:  
modified: `.idea/compiler.xml` and `.idea/misc.xml`  
deleted: `.idea/modules.xml` and `.idea/runConfigurations.xml`
(I see now that they're in the git repo so sorry if you intended for contributors to commit the changes)  
untracked/new: `.idea/jarRepositories.xml`  
I haven't done a lot of Android dev so I'm not exactly sure which are intended to be committed and which aren't.

flackbash commented:
You were right not to commit those :)
I should probably add them to the gitignore, I just want to do more research on it.

flackbash commented:
Merged and will be in the next release :)
